config/diaspora.yml.example

```diff
@@ -324,6 +324,22 @@ configuration: ## Section
         ## The debug level logs all XML sent and received by the server.
         #level: 'info'
 
+  ## Displays the location of a post in a map. Per default we are using the map
+  ## tiles of OpenMapSurfer (http://korona.geog.uni-heidelberg.de/).
+  ## If you want to use them you only have to enable the map. You also have the
```

Comment:
I think this is either misleading or not what we want, that is we don't want a config option to turn off the feature completely, just one to switch from the default to mapbox.